SCHEMAS.sql

```diff
@@ -0,0 +1,304 @@
+"SCHEMA FOR BOOKMYSHOW"
+
+CREATE TABLE USERS(
+    USERID INT IDENTITY(1,1) PRIMARY KEY ,
+    NAME VARCHAR(50),
+    EMAIL VARCHAR(50) UNIQUE,
+    PHONENUMBER VARCHAR(13),
+    PASSWORD VARCHAR(12),
+    REGISTRATIONDATE DATETIME DEFAULT CURRENT_TIMESTAMP
+  );
+
+CREATE TABLE MOVIES(
+    MOVIEID INT IDENTITY(1,1) PRIMARY KEY,
+    MOVIENAME VARCHAR(200),
+    MOVIEGENRE VARCHAR(100),
```

Comment:
Not Normalized

Reply:
Updated: separate tables are made for genres and language